lib/Sema/CSApply.cpp

```diff
@@ -109,7 +109,7 @@ Solution::computeSubstitutions(GenericSignature sig,
                               lookupConformanceFn);
 }
 
-static ConcreteDeclRef generateDeclRefForSpecializedCXXFunctionTemplate(
+static ValueDecl *generateDeclForSpecializedCXXFunctionTemplate(
```

Comment:
I know I named this function, but it's kind of ugly, so while you're updating the signature maybe change it to `generateSpecializedCXXFunctionTemplate`? WDYT?